examples/app.py

```diff
@@ -31,10 +31,10 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
-        pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
-        pip install invenio-assets==1.0.0a4
+        $ cdvirtualenv src/invenio-marc21
+        $ pip install -e .[all]
+        $ pip install "invenio-theme>=1.0.0a9"
+        $ pip install "invenio-assets>=1.0.0a4"
```

Comment:
> should always work with current version

I fully agree, however the example app is currently not CI-tested.  If it were, then the upper version limit wouldn't be needed.  

In other words, we could address my concern simply by generalising the example app testing, as we did here:

- https://github.com/inveniosoftware/invenio-records/pull/90

Note that problems already started to appear in other packages:

-  https://github.com/inveniosoftware/invenio-accounts/issues/115

> <=1.1

I thought about '<1.1' following semantic versioning.  This would be akin to saying: "The example app is not tested during CI, but I tested it manually, and it works well with `foo-1.7.x`, `bar-2.3.y`".